bin/src/main/resources/templates/index.html

```diff
@@ -37,10 +34,10 @@
     <option value="1">Sunday</option>
     <option value="2">Monday</option>
     <option value="3">Tuesday</option>
-    <option value="3">Wednesday</option>
-    <option value="3">Thursday</option>
-    <option value="3">Friday</option>
-    <option value="3">Saturday</option>
+    <option value="4">Wednesday</option>

```

Comment:
This looks a bit more distinct/unique.  :)   I recall this coming up in the last code review, as well.